src/vocutouts/handlers/external.py

```diff
@@ -149,8 +149,8 @@ async def _sync_request(
 
     # Redirect to the URL of the first result.
     butler = uws_factory.create_butler()
```

Comment:
In almost every other place one sees a `butler` variable they expect it to be a `lsst.daf.butler.Butler`, especially as the result of something called `create_butler`, but this doesn't seem to be a `Butler`.

Reply:
Yeah, good call, I've renamed it to `ResultStore`.